src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk

```diff
@@ -27,9 +27,15 @@
 
 {% macro paidInFullClaimantClaimSettledDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ defendantName }} paid you {{ amount }} on {{ moneyReceivedOn }}.',
-    { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral,
-      moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+    {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
```

Comment:
this won't work if there as no response. It is possible for the claim to not be responded and the claimant to still do paid in full. I would say you should check for if there is an amount in the response if there is not then just show the totalAmountTillToday and do some test scenarios around paid in full from different claim states

Reply:
Ok good point